.drone/drone.jsonnet

```diff
@@ -282,7 +282,20 @@ local manifest(apps) = pipeline('manifest') {
       {
         name: 'shellcheck',
         image: 'koalaman/shellcheck-alpine:stable',
-        commands: ['apk add make bash && make lint-scripts'],
+        commands: ['apk add make bash && make lint-scripts | tee -a ci.log'],
+      },
+      {
+        name: 'push logs',
+        image: 'grafana/promtail:2.0.0',
+        environment: {
+          GRAFANA_PUBLISHER_KEY: {
+            from_secret: 'lokibigtabledevuscentral0_publisher_token',
+          },
+          GRAFANA_USER: {
+            from_secret: 'lokibigtabledevuscentral0_user',
+          },
+        },
+        commands: ['cat ci.log | promtail --stdin --config.file promtail.yaml -config.expand-env=true'],
```

Comment:
I'm not really convinced. Alternatively, we could pipe the steps with `| tee >(promtail --stdin --config.file..)`